bots/pr/src/main/java/org/openjdk/skara/bots/pr/MergePullRequestReviewConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Remove 2019?

Reply:
No, the copyright lines from Oracle are either on the form `firstYear,` or `firstYear, lastYear,`. Never remove the first year.

Reply:
Thanks, fixed in latest commit!